storage/innobase/row/row0mysql.cc

```diff
@@ -2174,6 +2192,90 @@ row_mysql_unfreeze_data_dictionary(
 	trx->dict_operation_lock_mode = 0;
 }
 
+/** Function restores btr_pcur_t, creates dtuple_t from rec_t,
+sets row_end = CURRENT_TIMESTAMP/trx->id, inserts it to a table and updates
+table statistics.
+This is used in UPDATE CASCADE/SET NULL of a system versioning table.
+@param[in]	thr	current query thread
+@param[in]	node	a node which just updated a row in a foreign table
+@return DB_SUCCESS or some error */
+UNIV_INLINE
+dberr_t
+row_update_versioned_insert(que_thr_t* thr, const upd_node_t* node)
```

Comment:
Use `inline` instead of `UNIV_INLINE`. If the whole signature fits in 79 columns or less, then do not break lines:
```c++
inline dberr_t row_update_versioned_insert(que_thr_t* thr, const upd_node_t* node)
```
It does not fit, but if we abbreviated `update` to `upd` or `versioned` to `vers`, then it would fit.

Reply:
Ok. I've tried to avoid words shortening because of this rule http://google.github.io/styleguide/cppguide#General_Naming_Rules

Reply:
Btw, `static` is better here. Compiler may inline function if it hasn't `inline` specifier. And it will be easier for compiler to do so if it will know a function has a file scope.